src/esp/gfx/PbrShader.h

```diff
@@ -185,28 +190,91 @@ class PbrShader : public Magnum::GL::AbstractShaderProgram {
      * will use the node's semantic ID
      */
     InstancedObjectId = (1 << 12) | ObjectId,
+
+    /**
+     * Has ClearCoat layer.
+     */
+    ClearCoatLayer = 1 << 13,
+    // TODO : possibly break out each layer's flags to a separate flags state
+    // machine?
```

Comment:
Just name them nicer, and no shorthands ;) With the suggested variant reduction I think it's not too bad.

Also I wouldn't add everyting under the sun, just what _actually_ gets implemented in the shader in the end. E.g., for `KHR_materials_transmission` there's not even proper support for correct depth-ordered transparency yet, and additionally `KHR_materials_volume` would need to be implemented with some kind of raytracing. It'll make sense to implement those once Habitat gets _some_ transparency support at least.